HouseBuilder_Exercise2/src/houseBuilerPackage/House.java

```diff
@@ -1,25 +1,33 @@
 package houseBuilerPackage;
-
+import java.util.ArrayList;
 public class House {
 	
-	private String bathroom;
-	private String bedroom;
+	private ArrayList<String> bathroom;
```

Comment:
Looks better. I recommend you to create de ArrayList here in the definition and not in the constructor. It will work anyway